assets/model_monitoring/components/src/model_data_collector_preprocessor/run.py

```diff
@@ -58,6 +46,31 @@ def mdc_preprocessor(
 
     # Data column is a list of objects, convert it into string because spark.read_json cannot read object
     table = table.convert_column_types({"data": mltable.DataType.to_string()})
+    return table
+
+
+def mdc_preprocessor(
+    data_window_start: str,
+    data_window_end: str,
+    input_data: str,
+    preprocessed_input_data: str,
+    extract_correlation_id: bool
+):
+    """Extract data based on window size provided and preprocess it into MLTable.
+
+    Args:
+        production_data: The data asset on which the date filter is applied.
```

Comment:
Could you please update this Args list? It doesn't seem to match.

Reply:
Thanks I will follow-up in a bit